pkgs/applications/science/logic/nusmv/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NuSMV";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://nusmv.fbk.eu/distrib/NuSMV-${version}-linux64.tar.gz";
+    sha256 = "1370x2vwjndv9ham5q399nn84hvhm1gj1k7pq576qmh4pi12xc8i";
+  };
+
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  installPhase = ''
+    mkdir $out
+    ls bin
+    install -m755 -D bin/NuSMV $out/bin/NuSMV
+    install -m755 -D bin/ltl2smv $out/bin/ltl2smv
+    cp -r include $out/include
+    cp -r share $out/share
+  '';
+
+  meta = with lib; {
+    description = "A new symbolic model checker for the analysis of synchronous finite-state and infinite-state systems";
+    homepage = "https://nuxmv.fbk.eu/pmwiki.php";
+    maintainers = with maintainers; [ mgttlinger ];
```

```suggestion
    maintainers = with maintainers; [ mgttlinger ];
    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
```

Add this always when the source is not human-readable code.

---

(Unfortunately github has no command for merely adding lines)

---

I didn't know that. Thanks!

---

This was added a couple of months ago, mostly for tracking binary-only releases.